src/wp-includes/class-wp-scripts.php

```diff
@@ -418,15 +465,17 @@ public function do_item( $handle, $group = false ) {
 	 * Adds extra code to a registered script.
 	 *
 	 * @since 4.5.0
+	 * @since 6.3.0 Addition of $standalone boolean parameter.
```

Comment:
```suggestion
	 * @since 6.3.0 Added `$standalone` boolean parameter.
```

Update the document as the core is used for the new parameter.